### ISSUE TITLE
low validation accuracy with official pretrained models

### ISSUE THREAD

**Issue by zb1439** [pytorch/examples#550]
I tried evaluate on imagenet val set with official pretrained vgg16_bn with the following code.
`python main.py -a vgg16_bn --pretrained --evaluate <SOMEPATH>/imagenet/`
The accuracy is poor, got top1 acc 1.054 and top5 acc 2.134.
However I tried to swap the name of the directory of train set and val set to re-evaluate acc on test set, in case there was anything wrong with my command.
I got top1 and top5 acc over 90, which seemed to be correct with train set. I'm really confused what's wrong with my evaluation on test set.

**Comment by tjulyz** [comment]
Hi, have you solved this problem? I got the same problem. Thank you!

**Comment by tmac1997** [comment]
> I tried evaluate on imagenet val set with official pretrained vgg16_bn with the following code.
> `python main.py -a vgg16_bn --pretrained --evaluate <SOMEPATH>/imagenet/`
> The accuracy is poor, got top1 acc 1.054 and top5 acc 2.134.
> However I tried to swap the name of the directory of train set and val set to re-evaluate acc on test set, in case there was anything wrong with my command.
> I got top1 and top5 acc over 90, which seemed to be correct with train set. I'm really confused what's wrong with my evaluation on test set.

Hey have you solved this problem? Hope for your reply.

**Comment by XGBoost** [comment]
I have met the same problem,  and I have no idea

**Comment by XGBoost** [comment]
I think I have solved the problem. As mentioned in README.md, you should follow this script,https://raw.githubusercontent.com/soumith/imagenetloader.torch/master/valprep.sh to move the image to a certain directory.
